macros/dune/schema.sql

```diff
@@ -2,7 +2,7 @@
   {{ adapter.dispatch('create_schema', 'dbt')(relation) }}
 {% endmacro %}
 
-{% macro default__create_schema(relation) -%}
+{% macro trino__create_schema(relation) -%}
```

Comment:
where did you see downstream impact from this prior to change here?

Reply:
This breaks databricks if we ever need to create a new schema (e.g., for local development).